### PR TITLE
checkstyle change:

### PR DESCRIPTION
**Pull request by madongfly**
1. Adds <property name="separateLineBetweenGroups" value="true"/> to CustomImportOrder to enfore blank line between imports groups.
2. Uses checkstyle 6.5, which fixed a bug of "CustomImportOrder checks import sorting according to ASCII order instead of case-insensitive alphabetical order".

### CODE REVIEW

File: okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java [hunk 1]
```diff
@@ -41,7 +41,6 @@
 import com.squareup.okhttp.internal.spdy.Http20Draft16;
 import com.squareup.okhttp.internal.spdy.Settings;
 import com.squareup.okhttp.internal.spdy.Variant;
-
```
Comment (ejona86) [inline okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java]: This is the newline that caused this PR when I said should remain present. Were you testing the checkstyle checker and forgot to undo this?

**Comment by ejona86** [comment]
@madongfly, other than the newline removal, LGTM

File: okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java (hunk 1 above)
Comment (madongfly) [inline okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java]: Exactly, reverted this file.

**Comment by ejona86** [comment]
@madongfly, still LGTM

**Comment by madongfly** [comment]
This actually got merged to grpc:master, I forgot "push --force" to my PR before pushing the change to grpc:master, maybe that's why it shows "Closed" in stead of "Merged".

**Comment by ejona86** [comment]
@madongfly, yeah, I'd believe that. It isn't a big deal, even if it happened on every PR.

**Pull request closed without merging**